Tests/XCStringsToolTests/__Snapshots__/GenerateTests/testGenerate.Substitution.swift

```diff
@@ -156,6 +162,12 @@ extension LocalizedStringResource {
     /// - Note: Using ``LocalizedStringResource.Substitution`` requires iOS 16/macOS 13 or later. See ``String.Substitution`` for an iOS 15/macOS 12 compatible API.
     internal struct Substitution {
         /// A string that uses substitutions as well as arguments
+        ///
+        /// ### Source Localization
+        ///
+        /// ```
+        /// %@! There are %lld strings and you have %lld remaining
+        /// ```
```

Comment:
It's probably overkill, but we benefit from pulling out the substitution too.

<img width="735" alt="Screenshot 2024-05-16 at 20 21 50" src="https://github.com/liamnichols/xcstrings-tool/assets/482871/84075937-0ad0-4d1c-ad01-ab4296dd9345">